core/integration/features/verified/ProbOfTrading_Refactoring_Test.feature

```diff
@@ -0,0 +1,147 @@
+
+    Then the order book should have the following volumes for market "ETH/MAR22":
+      | side | price | volume |
+      | sell | 991   | 102    |
+      | sell | 990   | 1      |
+      | buy  | 900   | 1      |
+      | buy  | 899   | 112    | 
+      | buy  | 1     | 1      |
+  
+    Then the market data for the market "ETH/MAR22" should be:
+      | trading mode            | supplied stake | target stake |
+      | TRADING_MODE_CONTINUOUS | 50000          | 374541       |
+
+  Scenario: 002, market starts with a low best bid price 1 (ProbTrading is large), and then best bid goes to 899, LP is distressed 
+
+    And the parties submit the following liquidity provision:
+      | id  | party  | market id | commitment amount | fee   | side | pegged reference | proportion | offset | lp type    |
+      | lp1 | party0 | ETH/MAR22 | 50000             | 0.001 | sell | ASK              | 500        | 1      | submission |
+      | lp1 | party0 | ETH/MAR22 | 50000             | 0.001 | buy  | BID              | 500        | 1      | amendment  |
+
+    And the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference  |
+      | party1 | ETH/MAR22 | buy  | 1      | 1     | 0                | TYPE_LIMIT | TIF_GTC | buy-ref-1  |
+      | party1 | ETH/MAR22 | buy  | 1      | 900   | 0                | TYPE_LIMIT | TIF_GTC | buy-ref-1  |
+      | party2 | ETH/MAR22 | sell | 3      | 900   | 0                | TYPE_LIMIT | TIF_GTC | sell-ref-1 |
+      | party2 | ETH/MAR22 | sell | 1      | 990   | 0                | TYPE_LIMIT | TIF_GTC | sell-ref-2 |
+
+    When the opening auction period ends for market "ETH/MAR22"
+    Then the auction ends with a traded volume of "1" at a price of "900"
+
+    Then the order book should have the following volumes for market "ETH/MAR22":
+      | side | price | volume |
+      | sell | 990   | 1      |
+      | sell | 901   | 112    |
+      | sell | 900   | 2      |
+      | buy  | 1     | 100001 |
+
+    And the insurance pool balance should be "0" for the market "ETH/MAR22"
+
+    Then the parties should have the following account balances:
+      | party  | asset | market id | margin    | general    |
+      | party0 | USD   | ETH/MAR22 | 86478646  | 4913471354 |
+
+    When the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference    |
+      | party3 | ETH/MAR22 | buy  | 20     | 899   | 0                | TYPE_LIMIT | TIF_GTC | party3-buy-1 |
+
+    Then the order book should have the following volumes for market "ETH/MAR22":
+      | side | price | volume |
+      | sell | 990   | 1      |
+      | sell | 901   | 112    |
+      | sell | 900   | 2      |
+      | buy  | 899   | 20     |
+      | buy  | 898   | 112    |
+      | buy  | 1     | 1      | 
+
+    # lp1 which is party0 is distressed, and bond account had been slashed 
```

Comment:
I don't see this happening (bond account slashed)

Reply:
issue resolved after merge develop branch